lib/common/util.sh

```diff
@@ -314,3 +314,20 @@ unzip_file() {
     )
 }
 export -f unzip_file
+
+# delete files in directory that start like the given file name
+# (useful to purge error directory when same file is eventually
+# successfully processed)
+# $1 - directory
+# $2 - file name (basename)
+delete_files_in_dir() {
+    local dir="$1"; shift
+    local file="$1"; shift
+
+    log_info "Removing files in '$dir' that start with '$file'"
+    if [ -d $dir ]; then
+        log_info "Removing files '$dir/${file}*'"
+        rm -f $dir/${file}*
```

Comment:
Could be worth adding a `-v` so that the names of files being deleted are also logged.